scripts/ci/cleanup.sh

```diff
@@ -76,7 +76,7 @@ function snapshot() {
   snapshot_items intf $ext "ip -o link show | awk -F': ' '{print \$2}' | cut -d '@' -f 1"
   snapshot_items ovsdb $ext "ovs-vsctl list-br"
   snapshot_items docker $ext "docker ps -a -q"
-  snapshot_items docker-images $ext "docker images -a -q"
+  #snapshot_items docker-images $ext "docker images -a -q"
```

Comment:
Better to delete

Reply:
will do once I will have something really working